2-functions/2-arguments/1-before.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+final class ReportGenerator
```

Comment:
are we going emphasize on `final`? 
if not lets remove it, otherwise the audience may  think they would also need to use it (that if they pay attention of cause :smile: )